staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation_test.go

```diff
@@ -1600,13 +1600,7 @@ func TestValidationExpressions(t *testing.T) {
 				// avoiding empty list error with min and max by appending an acceptable default minimum value
 				"([0] + self.emptyInts).min() == 0",
 
-				// handle CEL's dynamic dispatch appropriately (special cases to handle an empty list)
```

Comment:
These should still be valid calls since they're valid CEL, and it would be good to preserve these cases in tests even if you remove the `dynSum` function that does dynamic dispatch.

Reply:
Updated